v3/src/components/map/utilities/rect-array.ts

```diff
@@ -0,0 +1,69 @@
+import {LatLngBounds} from "leaflet"
+
+export class RectRecord {
```

Comment:
`RectArray` and `RectRecord` seem unnecessarily vague. Perhaps something like `LatLngGrid` and `LatLngGridCell`?